SHARDING.md

```diff
@@ -0,0 +1,254 @@
+# Sharding Playbook
+
+This playbook walks through the steps to create a new log shard.
+The steps should be the same for Rekor and the CT log, though the
+CT process for generating key material may differ.
+
+Roughly, the steps are:
+
+1. Create the GCP resources with Terraform
+2. Generate key material
+3. Spin up the new infrastructure
+6. Update prober
+4. Perform manual testing
+5. Update TrustedRoot with the new key material and SigningConfig with the new shard URL
+6. Turning down the old shard
+
+## Create GCP resources
+
+New shards are declared as additional modules under the respective
+environment. This is a different pattern than other modules, which are all
+initialized through the `sigstore` module. Declaring each shard as its
```

Comment:
I'm not sure exactly what this means (probably because I have no context into infra). Why are other modules initialized through the single `sigstore` module?